src/app/searchResults/searchResults.tpl.html

```diff
@@ -14,7 +14,7 @@ <h1 ng-if="!$ctrl.searchParams.type || $ctrl.searchParams.type === 'people'">
 
       <div class="row">
         <div class="col-xs-12 col-sm-10 col-sm-offset-1">
-          <form method="get" action="/search-results.html">
+          <form method="get" action="">
```

Comment:
That's fine. I didn't really look to see if the html form submit was still being used.